src/MainPage/MainPage.js

```diff
@@ -80,23 +80,27 @@ function MainPage() {
       <div className='play-container'>
 
         
-      {pokemonData && pokemonData.data ? (
+ 
+        
+        <div className={`play-area-${Math.round(Math.random() * bgArray.length)}`} >
+
+        {pokemonData && pokemonData.data ? (
```

Comment:
I get whats happening in this line, it might be something to refactor down the line, calling pokemonData.data seems a little smelly